### DISCUSSION TITLE
compiler-macro-let

### DISCUSSION THREAD

**Discussion by fstamour** [fstamour/cl-car#8] [Ideas]
> There should be compiler-macro-let or similar.

**Comment by Gleefre** [comment]
~~Do you mean something like `compiler-let` from CLTL2?
Or is this more like `macrolet`/`flet` but for compiler-macros (that are defined via the `define-compiler-macro`)?~~

EDIT: On a second thought it reads only like a lexical binding for compiler-macros.
